src/samples/cover-list/v1/first-page.json

```diff
@@ -88,6 +88,278 @@
+                        "alt": "",
+                        "attribution": [
+                            "By some person."
+                        ],
+                        "source": {
+                            "mediaType": "image/jpeg",
+                            "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+                            "filename": "an-image.jpg"
+                        },
+                        "size": {
+                            "width": 4194,
+                            "height": 4714
+                        }
+                    },
+                    "thumbnail": {
+                        "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif",
+                        "alt": "",
+                        "source": {
+                            "mediaType": "image/jpeg",
+                            "uri": "https://iiif.elifesciences.org/lax/09560%2Felife-09560-fig1-v1.tif/full/full/0/default.jpg",
+                            "filename": "an-image.jpg"
+                        },
+                        "size": {
+                            "width": 4194,
+                            "height": 4714
+                        }
+                    }
+                },
+                "selectedCurator": {
+                    "id": "pjha",
+                    "type": {
+                        "id": "senior-editor",
+                        "label": "Senior Editor"
+                    },
+                    "name": {
+                        "preferred": "Prabhat Jha",
+                        "index": "Jha, Prabhat"
+                    }
+                },
+                "curators": [
+                    {
+                        "id": "pjha",
+                        "type": {
+                            "id": "senior-editor",
+                            "label": "Senior Editor"
+                        },
+                        "name": {
+                            "preferred": "Prabhat Jha",
+                            "index": "Jha, Prabhat"
+                        }
+                    }
+                ],
+                "content": [
+                    {
+                        "type": "research-article",
+                        "status": "poa",
+                        "id": "14107",
+                        "version": 1,
+                        "doi": "10.7554/eLife.14107",
+                        "authorLine": "Yongjian Huang et al.",
+                        "title": "Molecular basis for multimerization in the activation of the epidermal growth factor",
+                        "stage": "published",
+                        "published": "2016-03-28T00:00:00Z",
+                        "statusDate": "2016-03-28T00:00:00Z",
+                        "volume": 5,
+                        "elocationId": "e14107"
+                    }
+                ]
+            }
```

Comment:
We should add a blog-article if there isn't one already also. A few examples of properties that should not be present in the item as it is just the content snippet.

Reply:
The blog article is already there. That's why I didn't add it